websites/T/TMDB/dist/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Unformatted

Reply:
Pending

Reply:
I don't understand what you mean there

Reply:
https://github.com/PreMiD/Presences/blob/f06269e9e1d2a0aaaefea08075dc73fb684f2316/websites/T/TMDB/dist/metadata.json